### DISCUSSION TITLE
Use or with and example

### DISCUSSION THREAD

**Discussion by ismailghedamsi** [supabase/supabase#6347] [Questions] [answered]
Hi, I was testing the queries examples from supabase reference but I couldn't understand the Use or with and example. How are we testing countries name equality when the select is made on cities table also in the and operation we are checking if the name of the country is France and New Zealand.
Is there a mistake in the example or I read the query in the wrong way?
Thank you

**Comment by GaryAustin1** [comment, marked as answer]
I think the example is in error.  The "and" can’t ever be true.
Also agree those are in country table.
